templates/ERValign.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# set -x
+
+_DEBUG=!{debug}
+
+logMsg() {
+    if [[ $# -lt 2 ]];then
+        printf "\nUsage: logMsg <error_type> msg\nwhere <error_type> is one of 'INFO', 'WARN', 'DEBUG', 'CMD', or 'ERROR'.\n"
+        exit 1
+    fi
+    case "$1" in
+        'CMD' )
+            printf "+ [%s]:\t[CMD]\t%s\n" "$(date)" "$2"
+            ;;
+        'DEBUG' | 'true' )
+            if [ $_DEBUG == 'on' ];then printf "[%s]:\t[%s]\t%s\n" "$(date)" "$1" "$2";fi
+            ;;
+        'WARN' | 'INFO' | 'ERROR' ) 
+            printf "[%s]:\t[%s]\t%s\n" "$(date)" "$1" "$2"
+            ;;
+    esac
+    if [[ "$1" == ERROR ]];then
+        exit 1
+    fi
+} 
+
+function usage() {
+    echo "Usage: ERValign.sh <-r1|--read1> SAMPLE_1.fastq.gz <-r2|--read2> SAMPLE_1.fastq.gz [-o|--output] results/SAMPLE [-c|--cpus] Ncpus [-l|--limit-ram] 35129075129 [-d|--debug {off|on}]\nA genome folder should be present in /genome"
```

Comment:
I believe this help message is inherited from a previous usage of this script. It does not apply to this nextflow version.

Reply:
Correct, I'll remove it